.github/ISSUE_TEMPLATE/support_request.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Question or Support request
+about: Any questions you might have.
+title: ''
+labels: invalid
+assignees: ''
+
+---
+
+**Do not use issues for support requests or general help queries**
+
+Please ask on one of our mailing lists, or in our Slack channel. See the "Ask a question" section of http://airflow.apache.org/community/
+
+Issues which are not bug reports or feature requests (i.e. "how do I get X working", or "How do I best do X?") will likely be closed without a response.
```

Comment:
What do you think about extending this template with a few slack channels? For example with troubleshooting, kubernetes etc. ?

Reply:
We can't make those links can we? Or at least not for people who aren't already on the network. And if they are they probably know about them anyway?

Reply:
That's true, I have no strong opinion on that. But speaking about the links, we have to remember to update the website too as we mention "Use our Jira to create a ticket."

Reply:
Yeah, that's on my list already. Going to tackle it after this.

Reply:
I think lets leave it out here, and we can add a bit more info in the website.